src/java.xml.crypto/share/classes/com/sun/org/apache/xml/internal/security/algorithms/SignatureAlgorithm.java

```diff
@@ -191,7 +191,9 @@ private static SignatureAlgorithmSpi getSignatureAlgorithmSpi(String algorithmUR
                 }
             }
 
-            return implementingClass.newInstance();
+            @SuppressWarnings("deprecation")
+            SignatureAlgorithmSpi result = implementingClass.newInstance();
+            return result;
```

Comment:
I am using IntelliJ and there is not such a warning. I chose this style because that exactly where the suppressed warning happens and I don't want a reader to search for it.

That said, I'm now thinking of updating the `newInstance()` call with `getDeclaredConstructor().newInstance()`, as suggested by the `@deprecated` section of `newInstance()`. I'll also update to 2021.